README.md

```diff
@@ -1,76 +1,24 @@
 <a name="readme-top"></a>
 
-<!--
-HOW TO USE:
-This is an example of how you may give instructions on setting up your project locally.
-
-Modify this file to match your project and remove sections that don't apply.
-
-REQUIRED SECTIONS:
-- Table of Contents
-- About the Project
-  - Built With
-  - Live Demo
-- Getting Started
-- Authors
-- Future Features
-- Contributing
-- Show your support
-- Acknowledgements
-- License
-
-OPTIONAL SECTIONS:
-- FAQ
+<!-- TABLE OF CONTENTS -->
 
-After you're finished please remove all the comments and instructions!
--->
 
-<div align="center">
-  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
-  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
-  <br/>
 
-  <h3><b>Microverse README Template</b></h3>
+<!-- PROJECT DESCRIPTION -->
 
-</div>
+## Hello Microverse
```

Comment:
[REQUIRED]As the previous reviewer requested [here](https://github.com/Munyabelden/helloSetup/pull/1#discussion_r1068293800), please add customized information on your Getting Started section headings.


[OPTIONAL] Please customized all the sections, replace them with your project and personal data. Please remove all the instructions targeting you. For example: "Describe how a new developer could make use of your project."